### PR TITLE
use packages from cumulus core if they don't exist on npm [CUMULUS-512]

### PR DESCRIPTION
**Pull request by scisco**
fixes CUMULUS-512

### CODE REVIEW

**Comment by ghost** [comment]
Can you please document the developer workflow for the case where a developer needs to make a change to cumulus that requires updating the integration tests as well.  What steps do I need to take to smoothly get from “everything is working on master in both repos” to “everything is working on master in both repos _with_ my changes.”

**Comment by scisco** [comment]
@yjpa7145 added here https://github.com/cumulus-nasa/cumulus-integration-tests/pull/17/commits/3540f641180a6c7e59b2f6180210fc6b540eefee

File: spec/parsePdr/ParsePdr.input.template.json [hunk 1]
```diff
@@ -31,13 +33,13 @@
     "StatusReport": {},
     "ParsePdr": {
       "provider": "{{$.meta.provider}}",
-      "collection": "{{$.meta.collection}}",
-      "bucket": "{{$.meta.buckets.internal}}",
+      "internalBucket": "{{$.meta.buckets.internal}}",
```
Comment (laurenfrederick) [inline spec/parsePdr/ParsePdr.input.template.json]: So this is changing back to bucket now?

Comment (scisco) [reply spec/parsePdr/ParsePdr.input.template.json]: I don't know. may be in the interest time we shouldn't change it back.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,37 @@ Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and
...
+- If a package exists on npm, the source code of the packages is installed from npm
+- if a package does not exists on npm yet, circleci installs it from the Cumulus repo
+- by default the packages are installed from the `master` branch of the Cumulus repo
+- You can specify the branch of the cumulus repo by:
+  - setting `CUMULUS_BRANCH` environment variable on circleci
+  - adding the `.cumulus_branch` file on the root of the integration repo and specifying the branch name
+  - `CUMULUS_BRANCH` takes precedent over `.cumulus_branch` file name
+
+## Using latest Cumulus Source code for local tests
+By default, the integration tests use latests Cumulus packages published to the NPM. To use the packages from the Cumulus repository, do the following:
```
Comment (laurenfrederick) [inline README.md:110]: Nice.

File: spec/parsePdr/ParsePdr.input.template.json (hunk 1 above)
Comment (laurenfrederick) [reply spec/parsePdr/ParsePdr.input.template.json]: Marc already has a PR out for it

Comment (laurenfrederick) [reply spec/parsePdr/ParsePdr.input.template.json]: I think he should merge then we should just change it back, then get this PR merged and do the release. From there we can talk about moving to the Cumulus repo, although reading your process it doesn't really seem that bad. Slightly more cumbersome than having it in the Cumulus repo.

**Review by laurenfrederick — APPROVED** [review]
Going to approve so you're not blocked, but the internalBucket should go back to bucket

File: spec/parsePdr/ParsePdr.input.template.json (hunk 1 above)
Comment (ghost) [reply spec/parsePdr/ParsePdr.input.template.json]: The `internalBucket` to `bucket` change has been made to the master branch of the cumulus repo.  The `CUMULUS-450` branch of this repo has also been updated with that change.

Comment (scisco) [reply spec/parsePdr/ParsePdr.input.template.json]: changed.

**Pull request merged**